src/DDoSX/HstsClient.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace UKFast\SDK\DDoSX;
+
+use UKFast\SDK\DDoSX\Entities\Hsts;
+use UKFast\SDK\DDoSX\Entities\HstsRule;
+
+class HstsClient extends Client
```

Comment:
nit: consistency - `use UKFast\SDK\Client as BaseClient;` and extend that, then needs basepath adding